modules/dists/ReplicatedDist.chpl

```diff
@@ -177,10 +177,10 @@ when the constructor encounters an error.
 
 */
 class ReplicatedDist : BaseDist {
+  var targetLocDom : domain(Locales.idxType);
```

Comment:
There's a description of the ReplicatedDist changes in the PR message now (provided by @benharsh)